examples/design-system-tutorial/index.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>Design System Tutorial</title>
+        <style>
+            :not(:defined) {
+                display: none;
+            }
+
+            body,
+            html {
+                min-height: 100vh;
+                padding: 0;
+                margin: 0;
+                --top-offset: calc(var(--design-unit) * 28px);
+            }
+            body {
+                background-color: var(--fill-color);
+                color: var(--neutral-foreground-rest);
+                font-family: var(--body-font);
+            }
+
+            #root {
+                min-height: 100vh;
+                display: grid;
+                grid-template-columns: 1fr 4fr;
+                gap: calc(var(--design-unit) * 1px);
+            }
+
+            .canvas {
+                padding-right: calc(var(--design-unit) * 2px);
+                margin-top: var(--top-offset);
+            }
+
+            .container {
+                display: flex;
+                flex-wrap: wrap;
+            }
+
+            demo-navigation fluent-tree-view {
+                margin-top: calc(var(--top-offset) + (var(--design-unit) * 2px));
+            }
+
+            fluent-demo-card {
+                grid-row: 2;
+            }
+
+            fluent-demo-card:first-child {
+                grid-column: 2;
+            }
+
+            fluent-demo-card:nth-of-type(2) {
+                grid-column: 3;
+            }
+        </style>
```

Comment:
Nit - could this be in a CSS file? I'd love to see that (this) commented as well.